seed/challenges/08-coding-interview-questions-and-take-home-assignments/coding-interview-data-structure-questions.json

```diff
@@ -2703,12 +2680,12 @@
       "title": "Directed and Undirected Graphs",
       "description": [
         "This challenge starts a series of challenges that will teach graph data structures and some algorithms involving graphs. You may be familiar with graphs with an x- and y-axis found in your math classes. However, here graphs mean a different thing.",
-        "The data structure of graphs are collections of things and the relationships or connections among them. Graphs are also known as networks.",
-        "When talking about graphs, the precise term for \"objects\" are nodes or vertices. Similarly, the precise term for \"connections\" is edges.",
-        "One example of graphs is a social network where the nodes are you and other people, and the edges are whether two people are friends with each other.",
-        "There are two major types of graphs: directed and undirected. Undirected graphs are graphs without any direction on the edges between nodes. Directed graphs, in contrast, are graphs with a direction in its edges.",
+        "The data structure of <dfn>graphs</dfn> are collections of things and the relationships or connections among them. Graphs are also known as networks.",
+        "When talking about graphs, the precise term for \"objects\" are <dfn>nodes</dfn> or <dfn>vertices</dfn>. Similarly, the precise term for \"connections\" is <dfn>edges</dfn>.",
+        "One example of graphs is a social network where the <dfn>nodes</dfn> are you and other people, and the edges are whether two people are friends with each other.",
+        "There are two major types of graphs: <dfn>directed</dfn> and <dfn>undirected</dfn>. Undirected graphs are graphs without any direction on the edges between nodes. Directed graphs, in contrast, are graphs with a direction in its edges.",
         "An example of an undirected graph could be a social network. The nodes are people and the edges are friendship. An example of a directed network could be the internet and web page links. Here, the nodes are web pages and the directed edges are links to other pages, which might not necessarily point the other way.",
-        "Instructions",
+        "<hr>",
```

Comment:
I'm pretty sure `<hr>` is preferred over a heading, since the latter invite people to just skip the text and work towards passing the tests rather than learning the theory behind things.